README.md

```diff
@@ -463,6 +473,15 @@ __AWESOME__ apps counter: __150__ 🎉
 - [x] [GitHub](https://github.com/hanjoongcho/aaf-easydiary)
 - [ ] Official page
 
+## Download Manager
+### Download Navi
```

Comment:
What is the practicality of this app?

Reply:
Isn't every browser now has built in Download Manager or am I something misunderstanding?